viite-UI/src/view/ProjectLinkLayer.js

```diff
@@ -26,6 +27,11 @@
       strategy: ol.loadingstrategy.bbox
     });
 
+    var calibrationPointLayer = new ol.layer.Vector({
+      source: calibrationPointVector
+    });
+    calibrationPointLayer.set('name','calibrationPointLayer');
```

Comment:
Is it possible to set the name in the vector properties?
Like name: 'calibrationPointLayer'